### ISSUE TITLE
Method to remove all event sources

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#948]
Originally reported on Google Code with ID 678
```
I would like to have a simple public method that knows how to call removeEventSource
on each of the objects in the eventSources array.

This allows me to make a simple call and clean up the calendar in preparation for a
new set of event sources.

Also it would be great to set an array of event sources after the calendar has been
inited, rather than adding srcs in a loop.

thx
```

Reported by `vnehess` on 2010-10-20 05:48:47

Imported with **16** stars.

**Comment by arshaw** [comment]
[Code complete](https://github.com/fullcalendar/fullcalendar/commits/v2.8.0-dev) and [docs written](https://github.com/fullcalendar/fullcalendar/commits/docs-2.0) but not yet released. Will update this thread when released.

**Comment by arshaw** [comment]
Released in [v2.8.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v2.8.0).

http://fullcalendar.io/docs/event_data/removeEventSources/
(called with no arguments)

**Comment by erikbaan** [comment]
@arshaw could you remove my mail address ? https://github.com/fullcalendar/fullcalendar/issues/948#issuecomment-132547457
Probably some github glitch caused it to expose in the past.
